### PR TITLE
Bakula 7 8

### PR DESCRIPTION
**Pull request by sapanshah**
There will be subsequent PRs which will add new changes. For an example of what to do with the sub applications please see the healthcheck.py that was added under services.

Reviewers: @KrisSiegel @ericjperry

### CODE REVIEW

File: bakula/bottle/configuration.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import os
+
+from bottle import Bottle
+
+logger = logging.getLogger(__name__)
+
+def bootstrap_app_config(app):
+    """Bootstraps the application configuration
+
+    Method will deserialzie a JSON file into a dictionary and will load that into
```
Comment (ericjperry) [inline bakula/bottle/configuration.py]: deserialize

**Comment by ericjperry** [comment]
Other than the spelling error, :+1:

File: __main__.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    healthcheck.app
+]
+
+
+
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Bakula server')
```
Comment (KrisSiegel) [inline __main__.py]: Since we have a way to do configuration do we not want host and port to default to a value in a default configuration file?

**Comment by KrisSiegel** [comment]
Other than my question :+1:

File: __main__.py (hunk 2 above)
Comment (sapanshah) [inline __main__.py]: I've got something in mind in for this, but its a bit of work. I'm going to merge this for now and send a subsequent PR.

File: bakula/bottle/configuration.py (hunk 1 above)
Comment (sapanshah) [inline bakula/bottle/configuration.py]: Fixed

File: __main__.py (hunk 2 above)
Comment (KrisSiegel) [inline __main__.py]: :+1:

**Pull request merged**